_pages/en/capabilities.md

```diff
@@ -386,82 +386,68 @@ InfoSec supplies development teams and operational teams with an ample number of
 
 #### 3.1.2. Indicators  
 
-- What is the purpose of the monitor and control phase in your team?  
-   a. To measure the performance of the new process after the execution phase  
-   b. To monitor and control the planning and initiating phases  
-   c. To monitor the project team  
-   d. To control the project team's performance  
-   e. None of the above/not applicable  
-
-- What are the main steps involved in planning and implementing a monitoring protocol in your team?  
-   a. Complete background tasks  
-   b. Develop objectives  
-   c. Design and implement management  
-   d. Design the monitoring methodology  
-   e. Implement monitoring as a pilot study  
-   f. Implement monitoring  
-   g. Report and use results  
-   h. None of the above/not applicable  
-
-- Which step in planning and implementing a monitoring protocol in your team takes the most time?  
-   a. Complete background tasks  
-   b. Develop objectives  
-   c. Design and implement management  
-   d. Design the monitoring methodology  
-   e. Implement monitoring as a pilot study  
-   f. Implement monitoring  
-   g. Report and use results  
-   h. None of the above/not applicable  
-
-- What instrumentation do you use to monitor applications running in production?  
-   a. Availability monitoring via pinging or synthetic transactions  
-   b. Performance monitoring of the servers and infrastructure  
-   c. Real user monitoring  
-   d. Multi-tier views of end-to-end transactions including processing and network combined  
-   e. None of the above/not applicable  
+1-7 (strongly agree to strongly disagree)  
+Ideal: Strongly agree  
+
+- In my team monitoring and controlling phase is mostly to track applications and infrastructure in order to guide business decisions  
```

Comment:
Not sure what this question is asking me to agree with?

Reply:
removing it then